examples/mutators/apply-setters/simple/fn-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-func-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image: gcr.io/kpt-fn/apply-setters:unstable
+    config.kubernetes.io/local-config: 'true'
```

Comment:
IIUC it only affects functions that warp a kustomize plugin.
Can you try to remove this line?